### ISSUE TITLE
Add JSON export

### ISSUE THREAD

**Issue by stennie** [paralect/robomongo#764]
There's a general suggestion that [import/export should be integrated](https://github.com/paralect/robomongo/issues/102), but more detailed/practical use cases would be helpful.

Would welcome feedback on what would be most useful for JSON export:
 - export from a collection
 - export from a `find()` query
 - export from an `aggregate()` query
 - export via UI

Currently the best alternative is to use the standard `mongoexport` command line tool.

**Comment by Volox** [comment]
+1 For a wrapper of the `mongoexport` command line.

It would be great to export the DB/collection/query, also using the mongoexport command it will be easier, just run the command based on the context:
* Right click on a collection issues: `mongoexport -d <selected database> -c <selected collection>`.
* Button and/or context menu (with "*export*") on the tab issues: `mongoexport -d <selected database> -c <selected collection>`. *The query is passed to the `-q` parameter of `mongoexport` so not all the commands are available*
* For all the other commands maybe a custom export will be needed. For example using `mongo --eval "<command>"`

**Comment by bgSosh** [comment]
+1 for better export features.  I love robomongo, but this is a weak spot.

**Comment by maxxx999** [comment]
+1 robomongo is great thx for the great work you do 
 
an json export would be highly appriciated

**Comment by ownagedj** [comment]
+1 export/import would be awesome

**Comment by achselschweisz** [comment]
+1 for this, too. Would make this great tool even better!

**Comment by nitinsurana** [comment]
I used to use umongo, my colleague suggested robomongo, tried it and liked it so kudos for that.
Unfortunately, it didn't last long, went back to using umongo because of the export feature.

+1 export the query response in the UI to a json file.

**Comment by acaciocenteno** [comment]
+1 A non-valid JSON text export is the program's achille's heel.

Everything else is great, but this misfeature is a huge problem.

**Comment by akuckartz** [comment]
JSON-LD please :smile:

**Comment by raidenace** [comment]
Unfortunately I moved to other tools because of precisely the lack of this feature...

**Comment by anorsich** [comment]
@raidenace Sometimes you have to make a hard choice. We glad that you've used Robomongo and hope to see you back one this feature is ready :) Things should go much faster now.

**Comment by raidenace** [comment]
@anorsich: I hope to back too. Will keep checking for updates, hopefully it will be in a release soon. I think this request has been around for a few years now, actually.

**Comment by anorsich** [comment]
@raidenace That's right! If you saw read our blogs posts during [campaign](https://robomongo.org/campaign) you probably aware of reasons :) Anyways we plan to implement this soon! 

P.S. Will let you know as soon as it ready!

**Comment by Gastonite** [comment]
+1

**Comment by nazneen84** [comment]
+1

**Comment by cristtopher** [comment]
+1

**Comment by eli-js** [comment]
+1

**Comment by tunglt1810** [comment]
+1

**Comment by magomedov** [comment]
+1

**Comment by ni-vo** [comment]
+1
